geotrek/flatpages/static/flatpages/tinymce/js/additional_tinymce_plugins.js

```diff
@@ -41,21 +41,35 @@ document.addEventListener("DOMContentLoaded", function () {
                     }
                 ],
                 initialData: {
-                    label: tinymce.activeEditor.selection.getContent(),
+                    label: defaultValues && defaultValues.label || tinymce.activeEditor.selection.getContent(),
+                    target: defaultValues ? !!defaultValues.target : false,
+                    link: defaultValues ? defaultValues.link.replace(window.location.origin, '') : ''
                 },
                 onSubmit: function (api) {
                     var data = api.getData();
                     if (!data.link || !data.label) {
                         return;
                     }
-                    var target = data.target ? ' target="_blank" rel="noopener noreferrer" ' : '';
-                    editor.insertContent('<a class="button-link"' + target + 'href="' + data.link + '">' + data.label + '</a>');
+                    if (element) {
+                        element.textContent = data.label;
+                        element.href = data.link.replace(window.location.origin, '');
+                        if (element.target) {
```

Comment:
You want to check the `target` attribute from `data`. Otherwise the edition has no effect, it takes back its previous value.

Reply:
Oh yeah good point! I fix it immedialty :+1:

Reply:
Done